crates/core/src/kernel/schema/mod.rs

```diff
@@ -2,6 +2,7 @@ use std::any::Any;
 
 pub mod cast;
 pub mod partitions;
+#[allow(clippy::module_inception)]
```

Comment:
Clippy does not like the repeated name per https://rust-lang.github.io/rust-clippy/rust-1.91.0/index.html#module_inception

But, changing this could break API usage so I just disabled the warning.